freqtrade/rpc/emc.py

```diff
@@ -0,0 +1,249 @@
+"""
+ExternalMessageConsumer module
```

Comment:
would it be reasonable to rename this file to `external_message_consumer.py`? Acronyms sometimes make it tough to find something when looking for filenames.

Reply:
:+1:

Reply:
I didn't do this originally because it would make some imports incredibly long